library/src/main/java/com/aws/jverify/JVerify.java

```diff
@@ -0,0 +1,22 @@
+package com.aws.jverify;
+
+import java.util.function.Function;
+
+public class JVerify {
+    public static void check(boolean condition) {
+    }
+
+    public static <T> void precondition(boolean condition) {
```

Comment:
nit: type parameter not needed?
```suggestion
    public static void precondition(boolean condition) {
```